src/spyglass/common/common_nwbfile.py

```diff
@@ -310,21 +310,22 @@ def get_abs_path(analysis_nwb_file_name):
         analysis_nwb_file_abspath : str
             The absolute path for the given file name.
         """
-        base_dir = Path(os.getenv("SPYGLASS_BASE_DIR", None))
-        assert (
-            base_dir is not None
-        ), "You must set SPYGLASS_BASE_DIR environment variable."
+        try:
```

Comment:
This seems like it will break people's existing setups if they have the environmental variable set?

Reply:
[This line](https://github.com/LorenFrankLab/spyglass/blob/master/src/spyglass/settings.py#L154) already handles the setting of dj.config stores according to whatever settings.py resolves the analysis dir to be. My goal there was to centralize all path handling logic to one place so that, here, we can just `from spyglass.settings import analysis_dir` without having to check validity